src/main.ts

```diff
@@ -7,8 +7,13 @@ import { AnnotateArgs } from './types'
 import { annotate } from './annotator'
 import { parse } from './parser'
 
+import { Rosetta } from 'pz-rosetta-ts/lib/asledgehammer/rosetta/Rosetta'
 
 const annotateFiles = async (options: AnnotateArgs) => {
+
+    const rosetta = new Rosetta();
+    rosetta.load('assets/rosetta');
```

Comment:
This throws an error if the directory is absent. I would like the annotation to still function without requiring this file structure.

Ultimately, I think it would be best to be able to supply an alternative Rosetta directory as a command-line argument, but I can handle that when I restructure the project. For now, wrapping the `load` call in a try-catch should suffice.